_data/other.yml

```diff
@@ -115,6 +115,12 @@ websites:
       - totp
     doc: https://www.contentful.com/faq/two-factor-authentication/
 
+  - name: Craigslist
+    url: https://craigslist.org/
+    img: craigslist.png
+    twitter: craigslist
+    facebook: Craigslist-74561112489
```

Comment:
Page returns 404 for me.

Reply:
Twitter: https://twitter.com/craigslist
Facebook: https://www.facebook.com/Craigslist-74561112489

Both work for me.